src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_after_failing_staticcallFiller.json

```diff
@@ -0,0 +1,77 @@
+{
+    "returndatasize_after_failing_staticcall" : {
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "0x0a00000000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "pre" : {
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance" : "0x00",
+                "code": "{ (seq (STATICCALL 60000 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0) (SSTORE 0 (RETURNDATASIZE)))}",
+                "nonce" : "0x00",
+                "storage" : {
+                    "0x00" : "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+                }
+            },
+            "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x6400000000",
+								"//code": "an invalid opcode",
+                "code" : "0xfd",
```

Comment:
it is not good to have sender with code. some clients check that this is impossible situation.

Reply:
Thanks for pointing that out.  You're referring to the distinction between 'simple' accounts and 'contract' accounts right?

It should be easy enough to modify the tests to accommodate for this.